### ISSUE TITLE
Body is hidden or ellipsized if summary uses `<big>`

### ISSUE THREAD

**Issue by Yutsuten** [dunst-project/dunst#1416]
### Issue description

Prior to version `1.12.0` I was using `height = 300`, but now I replaced it with `height = (0, 300)` expecting it to behave the same, but it is not.
The issue is fixed if I also update my `format` setting by removing the `<big>` tag.

```diff
-format = "<small>%a</small>\n<big><b>%s</b></big>\n%b"
+format = "<small>%a</small>\n<b>%s</b>\n%b"
```

#### If using `<big><b>%s</b></big>` in format

1) Long body becomes elipsized:

```shell
notify-send -a 'Mail' -i mail-unread-symbolic 'New email' 'Dunst update is available! Check it now.'
```

![2024-12-14-17-47-28_664712066-full](https://github.com/user-attachments/assets/9d31fdc1-7fe8-48f3-ae5e-9458912c9bc2)

2) Shorter body becomes hidden:

```shell
notify-send -a 'Mail' -i mail-unread-symbolic 'New email' 'Dunst update is available!'
```

![2024-12-14-17-48-28_935550522-full](https://github.com/user-attachments/assets/da0540ac-ba15-4f43-8fb6-fb3ad5d2de76)

#### If using `<b>%s</b>` in format

Works as expected without `<big>`.

```shell
notify-send -a 'Mail' -i mail-unread-symbolic 'New email' 'Dunst update is available! Check it now.'
```

![2024-12-14-17-47-00_200662745-full](https://github.com/user-attachments/assets/b8ea744a-e069-4c51-9152-7a981d8071dc)

```shell
notify-send -a 'Mail' -i mail-unread-symbolic 'New email' 'Dunst update is available!'
```

![2024-12-14-17-48-51_559469465-full](https://github.com/user-attachments/assets/dd381288-1106-4e93-a21c-22e00665048f)

### Installation info

- Version:

```txt
Dunst - A customizable and lightweight notification-daemon 1.12.0 (2024-11-30)
Compiled on 2024-12-07 with the following options:
X11 support: enabled
Wayland support: enabled
SYSCONFDIR set to: /etc
Compiler flags: -g -std=gnu11 -pedantic -Wall -Wno-overlength-strings -Os -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions -Wp,-D_FORTIFY_SOURCE=3 -Wformat -Werror=format-security -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -g -ffile-prefix-map=/build/dunst/src=/usr/src/debug/dunst -flto=auto -pthread -MMD -MP
Linker flags: -lm -lrt -Wl,-O1 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,-z,pack-relative-relocs -flto=auto -lgio-2.0 -lgdk_pixbuf-2.0 -lpangocairo-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lharfbuzz -lcairo -lwayland-client -lwayland-cursor -lX11 -lXinerama -lXext -lXrandr -lXss
```

- Install type: `pacman`
- Window manager / Desktop environment: `Sway`
- Distro: `Arch Linux`

<details>
<summary>Minimal dunstrc</summary>

```ini
[global]
# Display
width = 350
height = (0, 300)
origin = top-right
offset = (35, 35)
indicate_hidden = yes
gap_size = 12
padding = 12
horizontal_padding = 20
frame_width = 1
sort = no

# Progress bar
progress_bar_frame_width = 0
progress_bar_corner_radius = 3

# Colors
foreground = "#cdd1dc"
frame_color = "#2d303c"
highlight = "#2274d5, #82aad9"

# Text
font = Noto Sans CJK JP 10
markup = full
format = "<small>%a</small>\n<big><b>%s</b></big>\n%b"
alignment = left
vertical_alignment = center
show_age_threshold = -1
hide_duplicate_count = false

# Icon
icon_position = left
min_icon_size = 54
max_icon_size = 80
icon_path = /usr/share/icons/Arc/status/96:/usr/share/icons/Arc/status/symbolic
icon_corner_radius = 4

[urgency_low]
background = "#383c4af0"
timeout = 0

[urgency_normal]
background = "#383c4af0"
timeout = 0

[urgency_critical]
background = "#9b4d4bf0"
highlight = "#eb4d4b"
timeout = 0

# Rules
[fullscreen_delay_everything]
fullscreen = delay

[fullscreen_show_critical]
msg_urgency = critical
fullscreen = show
```
</details>

**Comment by bynect** [comment]
it should be fixed with #1411 . If it is not please let me know because I am about to release 1.12.1 with the aforementioned fix

**Comment by Yutsuten** [comment]
Oh I may have a try before your release, I just have to see how to temporarily replace my system's dunst with the one in `master`.

**Comment by bynect** [comment]
you can clone dunst repo from git. then compile with `make all`. then kill the system dunst (pkill dunst). note if you are using systemd you may have to disable temporarily dunst `systemctl disable dunst`. Then just run `./dunst` from the repo

**Comment by Yutsuten** [comment]
Oh nice, it is indeed fixed in `master` ! Sorry for not noticing #1411 before creating the issue...

**Comment by bynect** [comment]
don't worry. thanks for checking  👍🏻
